src/common/dependencies.ts

```diff
@@ -1,13 +1,8 @@
-import semver from 'semver';
 import { Version } from './common-types';
-import { isArmMac, isMac, isWindows } from './env';
-
-const KB = 1024 ** 1;
-const MB = 1024 ** 2;
-const GB = 1024 ** 3;
 
 export interface PyPiPackage {
-    packageName: string;
+    displayName?: string;
+    pypiName: string;
     version: Version;
     findLink?: string;
```

Comment:
This type is slightly wrong, probably from before. The link can be string or null, not string or undefined.

```suggestion
    findLink?: string | null;
```